network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -237,7 +237,7 @@ def exec(self):
         return QueryAnswer(False, 'No redundancy found in ' + self.config.name)
 
     def compute_query_output(self, query_answer):
-        return self.get_query_output(query_answer, query_answer.bool_result)
+        return self.get_query_output(query_answer, add_explanation=query_answer.bool_result)
```

Comment:
same explanation of line 135 above.
an example:
query : redundantPoliciesInDenyFirstAlsoEgressDefaultAllow  file : tests/calico_testcases/example_policies/testcase11/testcase11-scheme.yaml

output before:

> Running query redundantPoliciesInDenyFirstAlsoEgressDefaultAllow
NetworkPolicy kube-system/testcase11-allow-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow
Ingress rule no. 1 in NetworkPolicy kube-system/testcase11-deny-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow
Egress rule no. 1 in NetworkPolicy kube-system/testcase11-default-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow

output after:

> Running query redundantPoliciesInDenyFirstAlsoEgressDefaultAllow
Redundancies found in np_denyFirstAlsoEgressDefaultAllow
NetworkPolicy kube-system/testcase11-allow-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow
Ingress rule no. 1 in NetworkPolicy kube-system/testcase11-deny-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow
Egress rule no. 1 in NetworkPolicy kube-system/testcase11-default-denyFirstAlsoEgress-defaultAllow is redundant in np_denyFirstAlsoEgressDefaultAllow